src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/InputWhitespace.java

```diff
@@ -85,6 +85,10 @@ private int nonVoid()
     private void testCasts()
     {
         Object o = (Object) new Object(); // ok
+        o = (java.lang.
+                Long)o; // error
+        o = (java.lang.
+                Long) o; // ok
```

Comment:
I was thinking about moving this case into separate method or file, but then I found a method with whitespace checks for typecasts. Going to move to a separate file.